.github/workflows/release-plugins.yaml

```diff
@@ -0,0 +1,157 @@
+name: Backstage plugin release
+
+on:
+  pull_request:
+    branches:
+      - 'release-*'
+    paths:
+      - 'backstage/plugins/knative-event-mesh-backend/**'
+  push:
+    branches:
+      - 'release-*'
+      # Do not run on push to release-next branch. We don't want to publish the plugin on every push to release-next.
+      # It might be okay to do so, but sobranch tool doesn't work well with release-next branch name.
+      - '!release-next'
+# Ideally, we would want to only publish the plugin on NPM when there's an actual code change in the plugin.
+# However, using the path filters block workflow execution on initial branch creation.
+# So, let's publish on every push to a release branch.
+# see https://github.com/openshift-knative/backstage-plugins/pull/15#issuecomment-2333324099
+#    paths:
+#      - 'backstage/plugins/knative-event-mesh-backend/**'
```

Comment:
This will increment the version on every push

Reply:
Have you seen this one? https://github.com/dorny/paths-filter?tab=readme-ov-file

Reply:
that combined with `github.event.created` https://github.com/openshift-knative/serverless-operator/blob/d5c4121801f91f2dbcaf3ccf4ae793e38266cd42/.github/workflows/bump.yaml#L16 seem to do it

Reply:
I am ok introducing that path filter.

Not sure if we need the `github.event.created` check.

The workflow was not triggered on branch creation due to top level path filters. Now that the top level filters are removed, it should be triggered. See https://github.com/orgs/community/discussions/26987

Reply:
> Not sure if we need the github.event.created check.

that is needed because the check becomes

```
if: ${{ github.event.created || steps.changes.outputs.src == 'true')
```